code/game/objects/effects/decals/cleanable/misc.dm

```diff
@@ -75,17 +75,35 @@
 	name = "glowing goo"
 	desc = "Jeez. I hope that's not for lunch."
 	icon_state = "greenglow"
+	var/lifetime = 240
 	light_power = 1
 	light_range = 2
 	light_color = LIGHT_COLOR_GREEN
 
 /obj/effect/decal/cleanable/greenglow/ex_act()
 	return
 
+/obj/effect/decal/cleanable/greenglow/Initialize()
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/effect/decal/cleanable/greenglow/process()
+	if(lifetime)
+		lifetime--
+	if(!lifetime)
+		name = "dried goo"
+		light_power = 0
+		light_range = 0
+		STOP_PROCESSING(SSobj, src)
+		update_light()
+	else
+		return
+
```

Comment:
```suggestion
	addtimer(CALLBACK(src, .proc/Decay), 24 SECONDS)

/obj/effect/decal/cleanable/greenglow/proc/Decay()
		name = "dried goo"
		light_power = 0
		light_range = 0
		update_light()

```

Reply:
Could you run me through why this is better than the solution I have

Reply:
doesn't run over process which could be used for something else,  also iirc process doesn't roll every tick so it is not as precise

Reply:
and also addtimer is basically meant for this